service-registry/src/main/java/io/servicecomb/serviceregistry/api/response/MicroInstanceResponse.java

```diff
@@ -0,0 +1,18 @@
+package io.servicecomb.serviceregistry.api.response;
+
+import io.servicecomb.serviceregistry.api.registry.MicroserviceInstance;
+
+public class MicroInstanceResponse {
```

Comment:
MicroserviceInstanceResponse is better?

Reply:
MicroInstanceResponse  change to MicroserviceInstanceResponse